node/popn.go

```diff
@@ -221,13 +220,30 @@ func New(ctx context.Context, opts Options) (*node, error) {
 		wallet.WithBLSSig(bls{}),
 	)
 
+	if opts.PrivKey != "" {
+		fmt.Println("==> Importing private keys...")
```

Comment:
If I'm not mistaken all the messages so far are feedback based on an operation that was executed, so to stay consistent we should only provide feedback after it's done like: `==> Imported private key %s` otherwise it's just noise in the terminal